keyboards/nopunin10did/kastenwagen/kastenwagen.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
This is a VIA json file, and not used by QMK. 

Please remove this.

And you want to PR this file to the `the-via/keyboards` repo, instead.

Reply:
Will do.